cookbooks/aws-parallelcluster-platform/resources/enroot/partial/_enroot_common.rb

```diff
@@ -33,34 +35,65 @@
     action :create_if_missing
   end
 
+  directory "#{node['cluster']['shared_dir']}/enroot" do
+    owner node['cluster']['cluster_user']
+    # group node['cluster']['cluster_user']
+    mode '1777'
+    action :create
+  end
+
+  directory "#{node['cluster']['shared_dir']}/pyxis" do
+    owner node['cluster']['cluster_user']
+    # group node['cluster']['cluster_user']
+    # mode '0755'
+    action :create
+  end
+
+  directory "/tmp/enroot/data" do
```

Comment:
this is not completely equivalent to what you removed from the script.
In the script also /tmp/enroot had perms 1777 not only /tmp/enroot/data